data/Andrewyithub.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "Andrew Y",
+  "bio": "Full Stack and Open Source. Travel & Fitness enthusiast. ✌️"
```

adding missing comma:
```suggestion
  "bio": "Full Stack and Open Source. Travel & Fitness enthusiast. ✌️",
```